### ISSUE TITLE
Google Maps - Bad performance, lags a lot!

### ISSUE THREAD

**Issue by NayamAmarshe** [brave/brave-browser#38241]
## Description 

Google maps lag a lot when zooming or panning. The issue is not fixed by turning off extensions either. Other browsers like Safari and Firefox work absolutely fine.

## Steps to Reproduce

   1. Open https://www.google.com/
   2. Zoom and Pan around
   3. The screen freezes occasionally.

## Actual result:

https://github.com/brave/brave-browser/assets/25067102/565f4087-f264-4e4d-a90e-44ab9a368e27

## Expected result:
Works fine on Firefox and Safari

https://github.com/brave/brave-browser/assets/25067102/e1a32099-23d4-49b4-9e24-b21bbee1bb68

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

1.65.132 Chromium: 124.0.6367.202 (Official Build) (arm64) 

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? Yes

## Miscellaneous Information:

It seems like Chromium has this issue. Not sure what the fix is. Google Chrome is lagging as well.

**Comment by rahulgoti-insight** [comment]
Sorry @NayamAmarshe Did you able to fix this?

**Comment by NayamAmarshe** [comment]
I don't know how to fix it tbh.

**Comment by rahulgoti-insight** [comment]
I'm facing the same problem on Brave.
[Version 1.65.133 Chromium: 124.0.6367.208 (Official Build) (x86_64)](https://brave.com/latest/)

**Comment by ShivanKaul** [comment]
@rahulgoti-insight can you check on Chrome? Is it better/worse than Brave for you?

**Comment by rebron** [comment]
Also @NayamAmarshe and @rahulgoti-insight Can you take a look at the latest version 1.66.110 and let us know if you're still seeing this performance issue?

**Comment by bsclifton** [comment]
This might be one where you need to go to brave://settings/system and disable hardware acceleration. @rahulgoti-insight @NayamAmarshe can you try that? I think the setting is now called `Use graphics acceleration when available`

**Comment by rahulgoti-insight** [comment]
Thanks @bsclifton for alternative solution :)

**Comment by NayamAmarshe** [comment]
> @rahulgoti-insight can you check on Chrome? Is it better/worse than Brave for you?

The issue is present on Chrome as well.

**Comment by NayamAmarshe** [comment]
> This might be one where you need to go to brave://settings/system and disable hardware acceleration. @rahulgoti-insight @NayamAmarshe can you try that? I think the setting is now called `Use graphics acceleration when available`

But I wouldn't like to disable hardware acceleration. It used to work fine a few month ago.

**Comment by NayamAmarshe** [comment]
> @NayamAmarshe good point - HW accel is massively useful and I wouldn't want to turn off either
> 
> I would like to recommend disabling HW acceleration only to see if that fixes the problem. **Then you can turn it back on**. If the problem experienced is related to HW acceleration, that helps us narrow it down. Then we might be interested in info from brave://gpu

Yes, disabling graphics acceleration does seem to fix the freezing issue but that might be because there's no smooth scroll happening.

**Comment by matyaspeto** [comment]
![image](https://github.com/user-attachments/assets/4a66dc2f-8d33-4da9-8bbe-3238df44dff2)

Clearing all kind of application data is fixed the issue for me. need to re-login after, but then it was smooth again.
